src/sphinx_book_theme/_transforms.py

```diff
@@ -0,0 +1,52 @@
+from sphinx.transforms.post_transforms import SphinxPostTransform
+from typing import Any
+from docutils import nodes as docutil_nodes
+from .nodes import SideNoteNode
+from sphinx.builders.html import StandaloneHTMLBuilder
+
+
+class HandleFootnoteTransform(SphinxPostTransform):
+
+    default_priority = 1
+
+    def apply(self, **kwargs: Any) -> None:
+        if isinstance(self.env.app.builder, StandaloneHTMLBuilder):
```

Comment:
No you just add `formats = ("html",)` (have a look at the sphinx post transform documentation)

Reply:
pretty neat. Thanks, have used that now.